src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -491,7 +491,7 @@ private function data_wp_interactive_processor( WP_Interactivity_API_Directives_
 		 * independently of whether the previous `data-wp-interactive` definition
 		 * contained a valid namespace.
 		 */
-		$new_namespace = null;
+		$new_namespace = 'WP';
```

Comment:
Can you clarify how this change fixes the problem? Difficult to understand from looking at the surrounding code, and it seems such a simple change. To be fair, I'm not very familiar with the underlying code :)

Reply:
How does this change impact accessing the context on the client?

Reply:
It should change the impact, rather than anyone can access namespaces not defined with this default one.

Reply:
> Can you clarify how this change fixes the problem? Difficult to understand from looking at the surrounding code, and it seems such a simple change. To be fair, I'm not very familiar with the underlying code :)

The directives were not being processed cause they expected non empty string as a namespace. While reading the `data-wp-interactive` attribute, we are returning a boolean in case is not defined `data-wp-interactive` or an empty string.

Adding a default one would fix it (my tests are running OK in local, not here 🤔 ), but it is not a good solution. That's why I wanted to check with @darerodz too 😅